cmd/deploy/remote.go

```diff
@@ -124,6 +128,16 @@ func newRemoteDeployer(builder *buildv2.OktetoBuilder) *remoteDeployCommand {
 }
 
 func (rd *remoteDeployCommand) deploy(ctx context.Context, deployOptions *Options) error {
+
+	sc, err := rd.clusterMetadata(ctx)
+	if err != nil {
+		return err
+	}
+
+	if deployOptions.Manifest != nil && deployOptions.Manifest.Deploy != nil && deployOptions.Manifest.Deploy.Image == "" {
+		deployOptions.Manifest.Deploy.Image = sc.PipelineRunnerImage
```

Comment:
Unless I'm missing something, I don't think this change is backwards compatible. With an old backend, `sc` will be `types.ClusterMetadata{}` and no image will be defined there.

Reply:
yup, i was asking this yesteday, what would the behavour be desired in this case. this is pending to solve right now

Reply:
updated.

>If the query doesn’t exist in the backend, we should fallback to a harcoded version of the images in the CLI
If the query returns an empty value for the images (because the backend is not returning them), we should fail